### PR TITLE
Wrap around long address string in employee directory pane

### LINKED ISSUE #188: [PE-D] Ui unable to display strings that are too long
When adding an address that is too long, the ui does not display the whole string.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Rogerlys/ped#3

### PR DESCRIPTION
**Pull request by yanbingtao**
- Wrap text when the address provided is too long to display in one line

### CODE REVIEW

File: src/main/resources/view/PersonListCard.fxml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
 
-<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (ureshiiYing) [inline src/main/resources/view/PersonListCard.fxml]: Change to version 11

**Review by ureshiiYing — COMMENTED** [review]
Overall, LGTM

File: src/main/resources/view/PersonListCard.fxml (hunk 1 above)
Comment (yanbingtao) [reply src/main/resources/view/PersonListCard.fxml]: Thanks! Will change to 11.

**Review by Diwu-Yi — APPROVED** [review]
LGTM

**Review by Persdre — APPROVED** [review]
LGTM

**Pull request merged**